src/components/index.ts

```diff
@@ -36,3 +36,4 @@ export { Tags } from './Tags';
 export { FloatingDropdown } from './FloatingDropdown';
 
 export { CheckboxGroup, CheckboxGroupProps, Layout };
+export { Popover } from './Popover';
```

Comment:
I think we can remove this file, since I don't think we should let users use this outside of the calendar for now.

Reply:
how will the calendar could use it than ?